src/Compilers/CSharp/Test/Syntax/Parsing/TopLevelStatementsParsingTests.cs

```diff
@@ -3597,5 +3597,426 @@ partial ext X
             }
             EOF();
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/67311")]
+        public void DelegateWithTupleReturnType_TopLevel()
+        {
+            var test = """
+                delegate int F(int x);
+                delegate int G((int, int) x);
+                delegate System.ValueTuple<int, int> H();
+                delegate (int, int) I();
```

Comment:
>delegate (int, int) I(); [](http://example.com/codeflow?start=16&length=24)

Consider splitting this test, a test per declaration.

Reply:
I'm ok with this.